ac/ac-train/src/Interface.jsx

```diff
@@ -0,0 +1,191 @@
+// @ flow
+import React from 'react';
+
+// UI
+import { isEqual } from 'lodash';
+
+import { Form, Command, DragDrop, Graphical } from './Interfaces';
+import Validation from './Validation';
+import { getCommandForTicket, generateTicket } from './ActivityUtils';
+
+const RunActivity = props => {
+  switch (props.activity) {
+    case 'command':
+      return <Command {...props} />;
+    case 'form':
+      return <Form {...props} />;
+    case 'dragdrop':
+      return <DragDrop {...props} />;
+    case 'graphical':
+      return <Graphical {...props} />;
+    default:
+      return <h1>Hello World</h1>;
```

Comment:
This is getting returned as one of the interfaces
<img width="534" alt="screen shot 2018-04-18 at 11 35 49 am" src="https://user-images.githubusercontent.com/2493446/38924979-a34318da-42fe-11e8-9c59-4181862ffcd9.png">